build/SharedFx.targets

```diff
@@ -10,6 +10,8 @@
 
     <FxProjectToBuild Condition=" '$(BuildSiteExtensions)' == 'true' " Include="$(RepositoryRoot)src\SiteExtensions\Runtime\Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj" />
     <FxProjectToBuild Condition=" '$(BuildSiteExtensions)' == 'true' " Include="$(RepositoryRoot)src\SiteExtensions\LoggingBranch\LoggingBranch.csproj" />
+
+    <FxProjectToBuild Condition=" '$(TargetRuntimeIdentifier)' == 'win-x86' " Include="$(RepositoryRoot)src\Framework\src\Microsoft.AspNetCore.App.Runtime.csproj" />
```

Comment:
This might cause a double-build. This project is already included here:
https://github.com/aspnet/AspNetCore/blob/337370263a7119ee5b8e0a6649d9e2da2fd9f0c3/build/repo.props#L149.  As long as this fixes the problem without double-building the project, this is fine with me as a short-term workaround.

The reason we are missing Microsoft.AspNetCore.App.Runtime.win-x86 is that #7832 moved the project from FxProjectToBuild to ProjectToBuild, which means `/t:BuildSharedFx` is not running the `Pack` target to run on Microsoft.AspNetCore.App.Runtime.csproj.

I would actually like to completely eliminate the `/t:BuildSharedFx` target, but we can leave that for another day.

Reply:
> I would actually like to completely eliminate the `/t:BuildSharedFx` target, but we can leave that for another day.

I completely agree. 

> This might cause a double-build. This project is already included here:

The condition should avoid the double build.